mkosi/__init__.py

```diff
@@ -1624,6 +1627,8 @@ def make_image(state: MkosiState, skip: Sequence[str] = [], split: bool = False)
     if not state.config.output_format == OutputFormat.disk:
         return []
 
+    normalize_mtime(state.root, state.config.environment.get("SOURCE_DATE_EPOCH"))
```

Comment:
We should call this before we call make_image(), and then fix the mtime's of any UKIs we install later individually.

Reply:
We tried to find all instances where files below state.root are modified and reset mtimes accordingly.